### PR TITLE
Update contact-pstn-service-desk.md

### PR DESCRIPTION
**Pull request by ErsancMS**
Modifying and adding extra information that will be useful when receiving new requests here at the Service Desk

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 1]
```diff
@@ -24,25 +24,30 @@ ms.custom:
...
-    - Include the Area Code(s)/Country/Region from which you would like telephone numbers.
-  - How many numbers you require (if this is for number acquisition).
-  - Authorizing person on the account and other important details based on your request.
+# PSTN Service Desk Contact
 
-Once you have all the information, send us your request to one the following email addresses that is applicable to your request:
-- **Number Porting**
-    - Porting telephone numbers **inside the United States or Canada**, [please send us an email at ptn@microsoft.com](mailto:ptn@microsoft.com).
-    - Porting telephone numbers **outside the United States or Canada**, [please send us an email at ptneu@microsoft.com](mailto:ptneu@microsoft.com).
+If you need assistance transferring/porting your existing phone numbers or acquiring new phone numbers for Microsoft Teams, please send us an email and include as many details as possible:
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md:30]:
```suggestion
If you need assistance with transferring/porting your existing phone numbers or acquiring new phone numbers for Microsoft Teams, please send us an email and include as many details as possible:
```

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 2]
```diff
@@ -24,25 +24,30 @@ ms.custom:
...
+  - Your [Organization ID (Tenant ID) or currently active Microsoft domains.]((https://docs.microsoft.com/en-us/onedrive/find-your-office-365-tenant-id)
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md:31]:
```suggestion
  - Your [Organization ID (Tenant ID) or currently active Microsoft domains]((https://docs.microsoft.com/onedrive/find-your-office-365-tenant-id).
```

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 3]
```diff
@@ -24,25 +24,30 @@ ms.custom:
...
+  - What types of phone numbers you need assistance with (**User/Subscriber** or **Service/Toll/Toll-Free**).
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md:32]:
```suggestion
  - What types of phone numbers you need assistance with (user/subscriber or service/toll/toll-free).
```

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 4]
```diff
@@ -24,25 +24,30 @@ ms.custom:
...
+  - Other important details based on your request.
+  - If you would like to **acquire new phone numbers** also include how many you require and the Area Code(s)/Country/Region from which you would like new phone numbers from.
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md:34]:
```suggestion
  - If you would like to acquire new phone numbers, also include how many you require and the Area Code(s)/Country/Region from which you would like new phone numbers.
```

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 5]
```diff
@@ -24,25 +24,30 @@ ms.custom:
...
     
 - **Number Acquisition**
-    - Acquiring new telephone numbers for any of the following countries: **Belgium, France, Germany, Great Britain, Ireland, Netherlands, Spain, Sweden, Switzerland**, [please send us an email at ptneu@microsoft.com](mailto:ptneu@microsoft.com)
-    - Acquiring new telephone numbers for the **Rest of the World** , [please send us an email at gcstnmsd@microsoft.com](mailto:gcstnmsd@microsoft.com).
+    - Acquiring new phone numbers for any of the following countries: **Belgium, France, Germany, Great Britain, Ireland, Netherlands, Spain, Sweden, Switzerland**, [please send us an email at ptneu@microsoft.com](mailto:ptneu@microsoft.com)
+    - Acquiring new phone numbers for the **Rest of the World**, [please send us an email at gcstnmsd@microsoft.com](mailto:gcstnmsd@microsoft.com).
+
+## Additional resources
 
+- To see _if_ and which **Calling Plans** are available in your country or region, read: [Countries and region availability for Audio Conferencing and Calling Plans](../country-and-region-availability-for-audio-conferencing-and-calling-plans/country-and-region-availability-for-audio-conferencing-and-calling-plans.md)
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md:47]:
```suggestion
- To see _if_ and which calling plans are available in your country or region, read [Countries and region availability for Audio Conferencing and Calling Plans](../country-and-region-availability-for-audio-conferencing-and-calling-plans/country-and-region-availability-for-audio-conferencing-and-calling-plans.md).
```

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 6]
```diff
@@ -24,25 +24,30 @@ ms.custom:
...
+- To assist you on choosing the correct type of phone numbers, read: [Different kinds of phone numbers used for Calling Plans](../different-kinds-of-phone-numbers-used-for-calling-plans.md).
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md:48]:
```suggestion
- To assist you in choosing the correct type of phone numbers, read [Different kinds of phone numbers used for Calling Plans](../different-kinds-of-phone-numbers-used-for-calling-plans.md).
```

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 7]
```diff
@@ -24,25 +24,30 @@ ms.custom:
...
+- To get assistance setting up phone numbers for your organization, read: [Support Contact for Business Products - Admin Help
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md:49]:
```suggestion
- To get assistance with setting up phone numbers for your organization, read [Support Contact for Business Products - Admin Help](https://docs.microsoft.com/microsoft-365/admin/contact-support-for-business-products?view=o365-worldwide&tabs=online)
```

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 8]
```diff
@@ -24,25 +24,30 @@ ms.custom:
...
+](https://docs.microsoft.com/en-us/microsoft-365/admin/contact-support-for-business-products?view=o365-worldwide&tabs=online).
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md:50]:
```suggestion
```

**Pull request merged**